src/app/entity-groups/research-entities/item-pages/person/person.component.ts

```diff
@@ -12,5 +25,81 @@ import { listableObjectComponent } from '../../../../shared/object-collection/sh
 /**
  * The component for displaying metadata and relations of an item of the type Person
  */
-export class PersonComponent extends ItemComponent {
+export class PersonComponent extends ItemComponent implements OnInit {
+
+  claimable$: BehaviorSubject<boolean> =  new BehaviorSubject<boolean>(false);
+
+  constructor(protected routeService: RouteService,
+              protected authorizationService: AuthorizationDataService,
+              protected notificationsService: NotificationsService,
+              protected translate: TranslateService,
+              protected researcherProfileService: ResearcherProfileService) {
+    super(routeService);
+  }
+
+  ngOnInit(): void {
+    super.ngOnInit();
+
+    this.authorizationService.isAuthorized(FeatureID.ShowClaimItem, this.object._links.self.href).pipe(
+      take(1)
+    ).subscribe((isAuthorized: boolean) => {
+      this.claimable$.next(isAuthorized);
+    });
+
+  }
+
+  claim() {
```

Comment:
This method needs TypeDocs & also needs specs added to `person.component.spec.ts`